packages/medusa-react/jest.config.js

```diff
@@ -1,19 +1,14 @@
 module.exports = {
-  // [...]
-  globals: {
-    "ts-jest": {
-      diagnostics: false,
-      isolatedModules: true,
-    },
-  },
   transform: {
-    ".(ts|tsx)$": require.resolve("ts-jest/dist/"),
+    ".(ts|tsx|ts)$": ["ts-jest", { isolatedModules: true, diagnostics: false }],
   },
   setupFilesAfterEnv: ["./jest.setup.js"],
   testEnvironment: "jsdom",
   moduleNameMapper: {
-    // Force module uuid to resolve with the CJS entry point, because Jest does not support package.json.exports. See https://github.com/uuidjs/uuid/issues/451
```

Comment:
suggestion: should we instead have `transformIgnorePatterns: ['/node_modules/'],` as far as I know, there is not reason to have to transform any node modules package right?